File: features/favorites/src/main/java/com/vp/favorites/model/ListItem.java
package com.vp.favorites.model;

import com.google.gson.annotations.SerializedName;

public class ListItem {
    public ListItem(){

    }

    public ListItem(String title, String year, String imdbID, String poster){
        this.title = title;
        this.year = year;
        this.imdbID = imdbID;
        this.poster = poster;
    }

    @SerializedName("Title")
    private String title;
    @SerializedName("Year")
    private String year;
    private String imdbID;
    @SerializedName("Poster")
    private String poster;

    public String getTitle() {
        return title;
    }

    public String getYear() {
        return year;
    }

    public String getImdbID() {
        return imdbID;
    }

    public String getPoster() {
        return poster;
    }
}

File: features/api/src/main/java/com/vp/api/model/ListItem.kt
package com.vp.api.model

import androidx.annotation.Keep
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity(tableName = ListItem.TABLE_NAME)
data class ListItem(
    @PrimaryKey @Keep val imdbID: String,
    @SerializedName("Title") val title: String,
    @SerializedName("Year") val year: String,
    @SerializedName("Poster") val poster: String?
) {

    companion object {
        const val TABLE_NAME = "favorite"
    }
}